backend/backend/urls.py

```diff
@@ -206,6 +206,7 @@
     path("api-auth/", include("rest_framework.urls", namespace="rest_framework")),
     path("api/", include(router.urls)),
     path("api/", include("core.urls")),
+    path("", include("core.urls")),
```

Comment:
now the core urls are in here twice. It'll work but it's a bit of a messy thing to do.

How about splitting the core urls up according to purpose? For example

```
    path("api/", include("core.api_urls")),
    path("admin/", include("core.admin_urls")),
```